src/components/DatePicker/index.tsx

```diff
@@ -0,0 +1,32 @@
+import { DatePicker, DatePickerProps } from '@mui/x-date-pickers/DatePicker';
+import React from 'react';
+import { Controller } from 'react-hook-form';
+
+type Props = DatePickerProps<string> & {
+  minDate: string;
+  maxDate: string;
+};
+const CustomDatePicker: React.FC<Props> = ({
```

Comment:
nazwę folderu też zmień na `dateInput`

```suggestion
const DateInput: React.FC<Props> = ({
```